includes/config/default.yml

```diff
@@ -45,10 +96,14 @@ default:
       paths:
         - %paths.base%/features
         - %paths.base%/../../includes/features
-    # Tests that are meant for development sites and can have config changes.
-    dev:
+
+    # --- Hosting Suites ---
+    # Tests for Stanford Sites hosting.
+    sites:
       filters:
-        tags: 'dev&&~live&&~prod'
+        tags: 'safe,sites&&~content&&~destructive'
```

Comment:
I like this idea but I would want to run ONLY sites and definitely nothing that could change anything.
I am thinking we can facilitate this by keeping our tag structure as is and creating an environment folder much like our sites and products folders to house the environment specific stuff.